govcd/lbappprofile_test.go

```diff
@@ -0,0 +1,75 @@
+// +build lb lbAppProfile functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_LBAppProfile(check *C) {
```

Comment:
comment would help here - what is testing

Reply:
Added.